Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -2065,6 +2077,31 @@ private extension Purchases {
         }
     }
 
+    #if DEBUG && !ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION
+    func runHealthCheckIfInForeground() {
+        // This is a workaround and needs to be fixed at some point. Explanation:
```

Comment:
@ajpallares I ended up having to add a workaround to this so that it does not run with the integration tests. Unfortunately the StoreKit tests rely too much on time and adding an extra requests makes products expire while requests are in flight. We really need to make the StoreKit integration tests more resilient, but this is a workaround to not make them more flaky for the time being.

Reply:
Let me know if you are okay with this, I have tried taking a stab at fixing the StoreKit tests flakiness and I got pretty far but wow they are incredibly fragile 😭